doc/topics/windows/windows-package-manager.rst

```diff
@@ -995,51 +975,46 @@ Firefox.
 
 .. note::
-    When storing software in the same location as the winrepo it is usually best
-    practice to place each installer in its own directory rather than in the
-    root of winrepo.
+    When storing software in the same location as the winrepo then
+    - Create a sub folder named after the package.
+    - Store the package definition file named ``init.sls`` and the binary installer in
+    the same sub folder if you are hosting those files on the ``file_roots``.
 
-    Best practice is to create a sub folder named after the package. That folder
-    will contain the software definition file named ``init.sls``. The binary
-    installer should be stored in that directory as well if you're hosting those
-    files on the file_roots.
-
-    ``pkg.refresh_db`` will process all ``.sls`` files in all sub directories
+.. note::
+    ``pkg.refresh_db`` command processes all ``.sls`` files in all sub directories
     in the ``winrepo_dir_ng`` directory.
 
 install_flags (str)
 -------------------
+The flags passed to the installer for silent installation.
 
-This setting contains any flags that need to be passed to the installer to make
-it perform a silent install. These can often be found by adding ``/?`` or ``/h``
-when running the installer from the command-line. A great resource for finding
-these silent install flags is the WPKG project wiki_:
+You can find these flags by adding ``/?`` or ``/h``when running the installer from the command-line.
+See `WPKG project wiki <https://wpkg.org/Category:Silent_Installers>`_ for information on silent install flags.
 
 .. warning::
-    Salt will appear to hang if the installer is expecting user input. So it is
-    imperative that the software have the ability to install silently.
+    Always ensure that the software has the ability to install silently since
+    Salt appears to hang if the installer expects user input.
 
 uninstaller (str)
 -----------------
 
-This is the path to the program used to uninstall this software. This can be the
-path to the same ``exe`` or ``msi`` used to install the software. Exe
-uninstallers are pretty straight forward. MSIs, on the other hand, can be
-handled a couple different ways. You can use the GUID for the software to
-uninstall or you can use the same MSI used to install the software.
+The path to the program to uninstall a software.
+This can be the path to the same  exe or msi used to install the software.
+If you use msi to install the software then you can either use GUID of the software or the
+same MSI to uninstall the software.
 
-You can usually find uninstall information in the registry:
+You can find the uninstall information in the registry:
 
 - Software\\Microsoft\\Windows\\CurrentVersion\\Uninstall
 - Software\\WOW6432Node\\Microsoft\\Windows\\CurrentVersion\\Uninstall
 
-Here's an example using the GUID to uninstall software.
+Here's an example of using the GUID to uninstall software.
```

Comment:
**Nitpick:** End this sentence with a colon (:) instead.

Reply:
Updated